### ISSUE TITLE
Provide example documentation

### ISSUE THREAD

**Issue by darthf1** [Kdyby/DoctrineDbalBatchImport#3]
Hi, can you provide example documentation how to use this?

**Comment by fprochazka** [comment]
I don't think that is a good idea to invest time in writing documentation, if I'm not happy with the code - https://github.com/Kdyby/DoctrineDbalBatchImport/issues/1. But it's definitely a thing I will have to do in the future.
